addon/tern/tern.js

```
@@ -0,0 +1,531 @@
+      if (dist < nearestDist) { nearest = from; nearestDist = dist; }
+    }
+    if (!nearest) return null;
+
+    if (before.length == 1)
+      nearest.ch += before[0].length;
+    else
+      nearest = Pos(nearest.line + (before.length - 1), before[before.length - 1].length);
+    if (data.start.line == data.end.line)
+      var end = Pos(nearest.line, nearest.ch + (data.end.ch - data.start.ch));
+    else
+      var end = Pos(nearest.line + (data.end.line - data.start.line), data.end.ch);
```

Biome (1.9.4)

[error] Shouldn't redeclare 'end'. Consider to delete it or rename it.

'end' is defined here:


(lint/suspicious/noRedeclare)